docs/async.rst

```diff
@@ -0,0 +1,38 @@
+===========
+Async/Await
+===========
+
+.. todo:: Fit this page into the walk-through sample story?
+
+Kopf supports the asynchronous handler functions::
+
+    import asyncio
+    import kopf
+
+    @kopf.on.create('zalando.org', 'v1', 'kopfexamples')
+    async def create_fn(spec, **_):
+        await asyncio.sleep(1.0)
+
+The async functions are a bit better than the regular functions,
```

Comment:
I'd rephrase this, to avoid making opinionated statements. E.g. "Async functions have the additional benefit over non-async ones to make the full stack trace available when exceptions occur or IDE breakpoints are used. "